src/AccordionSection/AccordionHeader.js

```diff
@@ -75,12 +90,18 @@ AccordionHeader.propTypes = {
   /**
    * Link variant for styling. See storybook for examples.
    */
-  variant: PropTypes.oneOf(VARIANTS).isRequired
+  variant: PropTypes.oneOf(VARIANTS).isRequired,
+
+  /**
+   * Optional flag to position icon to right of header text
+   */
+  chevronRight: PropTypes.bool
```

Comment:
I wonder if this would be better as something like `chevronPosition` that can be "left" or "right" (default left), but now I'm probably overthinking it. 😄  Feel free to ignore this.